jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -136,6 +136,10 @@ def query_args(self):
 
         return args
 
+class MediaRunner(BaseRunner):
+
+    def __init__(self, *args, **kwargs):
+        BaseRunner.__init__(self, *args, **kwargs)
```

Comment:
No need to add `__init__` just put `pass` in the next line after the class definition.